src/use_case/switch_view/SwitchViewUserDataAccessInterface.java

```diff
@@ -0,0 +1,5 @@
+package use_case.switch_view;
+
+public interface SwitchViewUserDataAccessInterface {
```

Comment:
Delete this file?

Reply:
Sure, yeah. Wasn't sure whether to leave it in or not.